### ISSUE TITLE
List of possible transitions

### ISSUE THREAD

**Issue by vaesbart** [rcarz/jira-client#68]
First of all : **thank you for creating this awesome project !**

I was trying out some things today, with our Jira install (v6.3.6a).
For example issue transistion; is it possible to get a list of available transitions for a given issue in a given state ?

For example : 

```java
for(Issue i : issues){
   logger.debug(String.format("Jira back To Do: %s", i));

   //Possible transitions ?
   final Issue.FluentTransition transition = i.transition();

   i.transition().execute("Stop Progress");
   i.refresh();
}
```

TIA
Bart.

**Comment by vaesbart** [comment]
Any info if this feature request is even being considered ?

**Comment by rcarz** [comment]
Yes, I think that's a reasonable request. Unfortunately I don't have a lot of time to work on this project. You're welcome to fork, make changes, and submit a pull request. Otherwise I do plan to spend some time soon working through the open issues.

**Comment by chaplinkyle** [comment]
I'll be submitting a pull request for this soon.

**Comment by vaesbart** [comment]
Thanks for the implementation :)

**Comment by rcarz** [comment]
fixed in #73
